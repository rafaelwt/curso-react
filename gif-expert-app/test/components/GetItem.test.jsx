const { render, screen } = require("@testing-library/react");
const { GifItem } = require("../../src/components/GifItem");

describe("Pruebas en GifItem", () => {
    const title = "Sample title";
    const url = "https://localhost/sample.jpg";
    test('debe hacer match con el snapshot', () => {
        const {container} = render(<GifItem  title={title} url={url} />);
        expect(container).toMatchSnapshot();
    });
    test('debe mostar la imagen con el url y alt indicado', () => {
        render(<GifItem  title={title} url={url} />);
        // screen.debug();
        
    });
});
