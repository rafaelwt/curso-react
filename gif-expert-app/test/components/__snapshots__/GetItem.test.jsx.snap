// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifItem debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Sample title"
      src="https://localhost/sample.jpg"
    />
    <p>
      Sample title
    </p>
  </div>
</div>
`;
